src/test/suite/StartIntegration.test.ts

```diff
@@ -66,18 +67,22 @@ suite('Check can deploy default examples', () => {
 		}
 		config.addNamespaceToConfig(undefined);
 	});
-
-	const testJava = test('Check can deploy Java example', async () => {
-		skipOnJenkins(testJava);
-		createdFile = await createFile(showQuickpickStub, showWorkspaceFolderPickStub, showInputBoxStub, 'TestBasicDeploy');
-
-		await startIntegrationWithBasicCheck(showQuickpickStub);
-	}).timeout(TOTAL_TIMEOUT);
+	
+	suite('Check basic deployments for each languages', function() {
+		LANGUAGES.forEach(function(language) {
```

Comment:
@lhein wasn't there an issue with using foreach and async operations? or am I imagining that?

Reply:
this is what official Mocha documentation is using https://mochajs.org/#dynamically-generating-tests

Reply:
I remember Lars mentioning that:
```
- `return` in a `forEach` loop doesn't return 😄  
   --> see https://stackoverflow.com/questions/2641347/short-circuit-array-foreach-like-calling-break
```
isn't this one? or it is something else that you have in mind?

Reply:
https://codeburst.io/javascript-async-await-with-foreach-b6ba62bbf404

Reply:
ok so we are not in this case. The direct function inside the foreach is not async